.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,6 @@
+echo "Post Create Starting"
+
+mvn install -B -V -ntp \
```

Comment:
This should be fine, since this is what we run in the first pre-submit step for this repository. It does a basic compilation of the spanner lib without any tests (thus verifying JDK compatibility).

Reply:
`mvn clean install` - Yes we can do this. I'll include